frontend/src/metabase/lib/formatting/colors.ts

```diff
@@ -1,21 +1,19 @@
 import { color } from "metabase/lib/colors";
 
-export function assignUserColors(
-  userIds: string[],
-  currentUserId: string,
-  colors = [
-    color("brand"),
-    color("accent2"),
-    color("error"),
-    color("accent1"),
-    color("accent4"),
-    color("bg-medium"),
-  ],
-) {
-  const assignments: { [index: string]: string } = {};
+const colors = [
```

Comment:
This breaks the custom color for enterprise when there's a custom color set
![image](https://user-images.githubusercontent.com/1937582/183571816-551d3a1d-80cb-420e-a456-6888cc2051f9.png)


#### Before
![Screen Shot 2022-08-09 at 12 29 00 PM](https://user-images.githubusercontent.com/1937582/183571715-7468d9fb-57b8-4b15-8930-146f7eaf2119.png)

#### This PR
![Screen Shot 2022-08-09 at 12 29 52 PM](https://user-images.githubusercontent.com/1937582/183571728-9eb7788b-9f3c-4dc1-8c2b-b7e4a2e63f2d.png)

The reason is that color is read from BE embedded in the HTML, but as `colors` is define as a constant here, this will be read before the color is updated by

https://github.com/metabase/metabase/blob/1140e62785ad6b08ec710dc46d2b318f96e4519c/enterprise/frontend/src/metabase-enterprise/whitelabel/lib/whitelabel.js#L164-L167

that's why it doesn't work.

The previous syntax works because [default parameter values are evaluated at call time](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#evaluated_at_call_time)

But I agree that removing the last param to indicate that it isn't passed is a good idea, so you could solve this by changing `colors` to a function instead to ensure that `color` will be evaluate after all the whitelabel colors are updated.

Reply:
@WiNloSt Nicely caught, Kelvin.

Updating, thanks for pointing this out.

Reply:
@WiNloSt I ended up reverting the colors to function arguments.
Extracting them had marginal gains at best, I believe.